cmd/limactl/shell.go

```diff
@@ -91,40 +108,45 @@ func shellAction(clicontext *cli.Context) error {
 	logrus.Debugf("changeDirCmd=%q", changeDirCmd)
 
 	script := fmt.Sprintf("%s ; exec bash --login", changeDirCmd)
-	if clicontext.NArg() > 1 {
-		script += fmt.Sprintf(" -c %q", shellescape.QuoteCommand(clicontext.Args().Tail()))
+	if len(args) > 1 {
+		tailArgs := args[1:]
+		script += fmt.Sprintf(" -c %q", shellescape.QuoteCommand(tailArgs))
```

Comment:
This may be just personal preference, but why introduce a temporary variable that is used only once, and is not even shorter than the expression it replaces?

```suggestion
		script += fmt.Sprintf(" -c %q", shellescape.QuoteCommand(args[1:]))
```